chaincfg/params.go

```diff
@@ -611,6 +623,97 @@ var TestNet3Params = Params{
 	SlpAddressPrefix:    "slptest",
 }
 
+// TestNet4Params defines the network parameters for the test Bitcoin network
+// (version 3).  Not to be confused with the regression test network, this
+// network is sometimes simply called "testnet".
+var TestNet4Params = Params{
+	Name:        "testnet4",
+	Net:         wire.TestNet4,
+	DefaultPort: "28333",
+	DNSSeeds: []DNSSeed{
+		{"testnet4.imaginary.cash", true},
+	},
+
+	// Chain parameters
+	GenesisBlock:  &testNet4GenesisBlock,
+	GenesisHash:   &testNet4GenesisHash,
+	PowLimit:      testNet3PowLimit,
```

Comment:
Should we have another variable for the PowLimit? Odd to use testNet3 values here.